scripts/paternity-test.py

```diff
@@ -0,0 +1,173 @@
+import io
+import os
+import json # might not be needed
+from firecloud import fiss
+from firecloud.errors import FireCloudServerError
+import firecloud.api as fapi
+import numpy as np
+import pandas as pd
+import logging
+#import re <-- doesn't work on Terra
+
+# User-set variables
+BILLING_PROJECT_ID="biodata-catalyst"
+WORKSPACE="TSV-AFY"
+SUBDIRECTORY="\/case2\/"
+TABLE_NAME="CRAMs_and_CRAIs" #Do not include spaces or weird characters
+PARENT_FILETYPE="cram"
+# If your filenames are like this, please set INCLUDE_PARENT_EXTENSION to True:
+# NWD119844.CRAM
+# NWD119844.CRAM.CRAI
+# If you filenames are like this, please set INCLUDE_PARENT_EXTENSION to False:
+# NWD119844.CRAM
+# NWD119844.CRAI
+# No quotation marks for this variable, just True or False
+INCLUDE_PARENT_EXTENSION = True
+# Make sure to escape the slashes in your SUBDIRECTORY variable. 
+# For instance, if your files are in a folder called "testfiles" you will need to
+# enter it as "/\testfiles/\" or else Python will get angry.
+
+# Call FireCloud API
+try:
+    bucket = os.environ["WORKSPACE_BUCKET"]
+    response = fapi.list_entity_types(BILLING_PROJECT_ID, WORKSPACE)
+    if response.status_code != 200:
+        print("Error in Firecloud, check your billing project ID and the name of your workspace.")
+    else:
+        print("Firecloud has found your workspace!")
+        directory = bucket + SUBDIRECTORY
+except NameError:
+    print("Caught a NameError exception. This probably means you didn't restart the kernal after"
+          " running the first block of code (the one with all the imports). Run it again, restart"
+          " the kernal, then try running every block of code (including the import one) again.")
```

Comment:
You probably want to halt execution here as well. This can be done by changing `print` to `Exception`, or adding a `raise` to trigger the original name error:
```
try:
    stuff()
except NameError:
    print("...")
    raise
```